src/promptflow/promptflow/batch/_batch_inputs_processor.py

```diff
@@ -51,12 +51,22 @@ def _resolve_input_data(self, input_dirs: Dict[str, str]):
     def _resolve_data_from_input_path(self, input_path: Path):
         """Resolve input data from directory"""
         result = []
+        max_rows_count: int = None
+        if self._max_lines_count:
+            # Add one more row to check if the data exceeds the limit.
+            max_rows_count = self._max_lines_count + 1
```

Comment:
You can modify the content of the log and change '>' to '>='

Reply:
Fixed. Thanks.